spk/comskip/Makefile

```diff
@@ -0,0 +1,38 @@
+SPK_NAME = comskip
+SPK_VERS = 0.81.089
+SPK_REV = 1
+SPK_ICON = src/comskip.gif
+
+DEPENDS = cross/ffmpeg cross/argtable cross/$(SPK_NAME)
+
+RELOAD_UI = yes
```

Comment:
Package Center will refresh itself, no problem. The RELOAD_UI is to refresh the DSM interface altogether. That includes animating the placement of the icon in the menu and the `1` showing up on the menu button to indicate a new package is installed.
Seeing as the package doesn't need that, this is unnecessary here.